bench-run/requirements.txt

```diff
@@ -0,0 +1,2 @@
+GitPython>=3.1
```

Comment:
Looks like it should be placed in the `publication` folder.

Reply:
moved